insuree/models.py

```diff
@@ -298,6 +298,23 @@ class Meta:
         managed = False
         db_table = 'tblInsuree'
 
+class InsureeAttachment(models.Model):
+    """ Class Attachment :
+    Class for isurees attachments
+    """
+    idAttachment = models.AutoField(
+        primary_key=True
+    )
+    folder = models.CharField(db_column='Folder', max_length=250, null=False)
+    name = models.CharField(db_column='FileName', max_length=250, null=False)
+    insuree = models.ManyToManyField('Insuree')
```

Comment:
Is there a case when an attachment is attached to multiple insurees?

Reply:
Fix to change cardinality Pushed